src/views/home/PlayerHome/components/activities/TutorialActivity.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <BaseActivity :heading="heading" :slideTo="level">
+    <template v-slot:extra>
+      <img
+        :src="progressImage"
+        class="player-progress-bar"
+        v-if="!$vxm.user.hasLabAccess"
+      />
+    </template>
+    <SwiperSlide v-for="(item, index) in tools" :key="index">
+      <QuestCard :key="item.title" v-bind="item" />
+    </SwiperSlide>
+  </BaseActivity>
+</template>
+
+<script lang="ts">
+  import {Vue, Component, Prop} from 'vue-property-decorator';
+  import { SwiperSlide } from 'vue-awesome-swiper';
+  import {RoadmapAchievement} from '@/types/common-types';
+  import Utils from '@/utils/utils';
+  import PROGRESS_IMAGE_0 from '@/assets/progress/progress0.svg';
+  import PROGRESS_IMAGE_1 from '@/assets/progress/progress1.svg';
+  import PROGRESS_IMAGE_2 from '@/assets/progress/progress2.svg';
+  import PROGRESS_IMAGE_3 from '@/assets/progress/progress3.svg';
+  import PROGRESS_IMAGE_4 from '@/assets/progress/progress4.svg';
+  import PROGRESS_IMAGE_5 from '@/assets/progress/progress5.svg';
+  import PROGRESS_IMAGE_6 from '@/assets/progress/progress6.svg';
+  import QuestCard from '@/components/Cards/QuestCard.vue';
+  import BaseActivity from './BaseActivity.vue';
+
+  const PROGRESS_IMAGES = [
+    PROGRESS_IMAGE_0,
+    PROGRESS_IMAGE_1,
+    PROGRESS_IMAGE_2,
+    PROGRESS_IMAGE_3,
+    PROGRESS_IMAGE_4,
+    PROGRESS_IMAGE_5,
+    PROGRESS_IMAGE_6,
+  ];
```

Comment:
code smell; Is there any way to do an array import (even if you specify the paths literally)?

Reply:
No - but what should really happen is that it should be a dynamic SVG. Will file a separate issue